packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -50,12 +52,13 @@ export class VaRadioOption {
       <Host
         aria-checked={this.checked ? `${this.checked}` : 'false'}
         aria-describedby={ariaDescribedby}
+        aria-label={this.name}
```

Comment:
Can we get this covered in a test to try to prevent regressions?

Reply:
@bkjohnson updated code has been pushed to include a test covering this here: https://github.com/department-of-veterans-affairs/component-library/blob/ae05082e1bd712b70fc3d6fd773321e84cf6e48d/packages/web-components/src/components/va-radio-option/test/va-radio-option.e2e.ts#L12